vue/src/core/components/LikeUnlikeButton.vue

```diff
@@ -2,7 +2,9 @@
   <b-button @click="$emit('onLikeClicked', item)" size="sm" pill :variant="liked ? 'info' : 'light'">
     <i class="far fa-thumbs-up fa-lg"/>
     {{ $t('Like') }}
-    <b-badge v-if="item" class="ml-2" pill variant="secondary">
+    <b-popover target="exPopover" container="special-popover" title="Prop Examples" triggers="hover focus" content="Embedding content using properties is easy">
```

Comment:
@Null-IO Does this popover need that properties?

Reply:
The whole b-popover tag is not needed actually. Sorry leftovers from debugging